File: CHAP -- @Notes/java/RecapsulationCodes/RecapsulationCodes/Encapsulation-DataHiding/PrivateDataMethod.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace RecapsulationCodes.Encapsulation_DataHiding
{
    class Encapsul
    {
        private int a, b, c;
        
        private void getVal()
	    {
		    Console.Write("Enter the vlaue of a: ");
		    a = int.Parse(Console.ReadLine());

		    Console.Write("Enter the vlaue of b: ");
		    b = int.Parse(Console.ReadLine());
	    }

	    public void multiplication()
	    {
		    getVal();
		    c = (a * b);
	    }

	    public void disp()
	    {
		    Console.WriteLine("Multiplication is: " + c);
	    }
    }

    class PrivateDataMethod
    {
        public static void Main(string[] args)
        {
            Encapsul enObj = new Encapsul();

            enObj.multiplication();
            enObj.disp();

            Console.WriteLine();
            Console.WriteLine("Please any key to continue...!");
            Console.ReadKey(true);
        }
    }
}

File: CHAP -- SWING (desktop application)/IIHT/Code/Source_Code (BlueJ)/SourceCode-JCreatorPro/PracticeCode/src/Encapsulation-DataHiding/PrivateDataMethod.java
import java.io.*;

class Encapsul
{
	private int a, b, c;

	InputStreamReader read = new InputStreamReader(System.in);
	BufferedReader in = new BufferedReader(read);

	private void getVal()throws IOException
	{
		System.out.print("Enter the vlaue of a: ");
		a = Integer.parseInt(in.readLine());

		System.out.print("Enter the vlaue of b: ");
		b = Integer.parseInt(in.readLine());
	}

	void multiplication()throws IOException
	{
		getVal();
		c = (a * b);
	}

	void disp()
	{
		System.out.println("Multiplication is: " + c);
	}
}

class PrivateDataMethod
{
	public static void main (String[] args)throws IOException
	{
		Encapsul enObj = new Encapsul();

		enObj.multiplication();
		enObj.disp();
	}
}